cloudwatch-readonly-user.tf

```diff
@@ -0,0 +1,52 @@
+data "aws_iam_policy_document" "cloudwatch_readonly_user_policy_document" {
```

Comment:
Not all permissions are related to cloudwatch. maybe cloudwatch user is not the best name. Should we rename it to monitor user?

Reply:
Renamed